### DISCUSSION TITLE
iFrame support

### DISCUSSION THREAD

**Discussion by Kaynoux** [seriousm4x/UpSnap#658] [Feature requests]
It would be nice if I could use UpSnap embedded as an iframe on other sites. Currently this is not possible.
> Refused to display '(URL)' in a frame because it set 'X-Frame-Options' to 'sameorigin'

**Comment by Kaynoux** [comment]
Found a workaround: I used nginx proxy manager to change the X-Frame config. It works after adding this line 
``proxy_hide_header X-Frame-Options;``

**Reply by ThePulpoLopez** [reply]
This was the real deal! thanks for sharing! was looking for this as I wanted to show upsnap directly on my homeassistant control panel and this was the only good response.
